Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2831,3 +2839,27 @@
     version: "2.0"
     engine: Blink
     engine_version: ""
+-
+  user_agent: Mozilla/5.0 (Linux) AppleWebKit/534.51 (KHTML, like Gecko) Ekioh/2.2.4.7-moto-mob Safari/534.51
+  client:
+    type: browser
+    name: Flow
+    version: 2.2.4.7
+    engine: EkiohFlow
+    engine_version: ""
+-
+  user_agent: Mozilla/5.0 (Linux) AppleWebKit/534.51 (KHTML, like Gecko) Ekioh/2.2.4.7-moto-mob Safari/534.51:534 Motorola KreaTV STB VIP1003
+  client:
+    type: browser
+    name: Flow
+    version: 2.2.4.7
+    engine: EkiohFlow
+    engine_version: ""
+-
+  user_agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_0) EkiohFlow/5.7.4.30559 Flow/5.7.4 (like Gecko Firefox/53.0 rv:53.0)
+  client:
+    type: browser
+    name: Flow
+    version: 5.7.4.30559
+    engine: EkiohFlow
+    engine_version: 5.7.4.30559
```

Comment:
Is that really correct? Based on their changes for which browser, I guess the browser version should be `5.7.4` only. See https://github.com/WhichBrowser/Parser-PHP/pull/487/files#diff-be0163865273a04242de6cf101ffcf4276132dffcf51b9e233e49f6f54234056

Reply:
@sgiehl to fix the engine version, need to add the code in Engine/Version.php
```php
if ('EkiohFlow' === $this->engine) {
      $pattern = '~EkiohFlow/(\d+\.\d+\.\d+)~i';

      if (\preg_match($pattern, $this->userAgent, $matches)) {
            return ['version' => \array_pop($matches)];
      }
}
```
adding?

Reply:
You mean in order to cut off the build release number?

Reply:
Yes

Reply:
Guess keeping the build release number might be fine as well. Guess this might be the same for other browser engines as well